data/presets/presets.json

```diff
@@ -8456,6 +8456,26 @@
             },
             "name": "Golf Course"
         },
+        "leisure/hackerspace": {
+            "icon": "commercial",
+            "fields": [
+                "name",
+                "address",
+                "website"
```

Comment:
You can add `"building_area"` and `"opening_hours"` here.. Actually I'd probably just copy whatever other indoor leisure presets have, like [`bowling_alley.json`](https://github.com/openstreetmap/iD/blob/master/data/presets/presets/leisure/bowling_alley.json)